system_reports/osx-xamarin-stable.log

```diff
@@ -61,7 +61,7 @@ Homebrew/homebrew-core (git revision 18dd; last commit 2017-03-17)
 * Rubygems: 2.5.2
 * Bundler: Bundler version 1.13.6
 * CocoaPods: 1.1.1
-* fastlane: fastlane 2.1.3
+* fastlane: [32m# fastlane 2.23.0 is available. You are on 2.1.3.[0m
```

Comment:
yeah, we might have to switch to get the version from a `gem` call instead of from `fastlane --version`, but in general we prefer to call version on the executable directly, to detect certain issues (when the gem is installed but the executable is not available in the PATH for example).